Source/platform/android/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+include(functions/devilutionx_library)
+add_devilutionx_object_library(libdevilutionx_android android.cpp)
+
+target_link_libraries(libdevilutionx_android PUBLIC
+  DevilutionX::SDL
+  SDL_audiolib::SDL_audiolib
+)

```

Thanks to the changes you made to remove duplicate code, `android.cpp` no longer includes `engine/assets.hpp` which was the reason why I had to link `SDL_audiolib`.
```suggestion
target_link_libraries(libdevilutionx_android PUBLIC DevilutionX::SDL)
```